SECURITY.md

```diff
@@ -0,0 +1,3 @@
+# OpenJDK Vulnerabilities
+
+Please follow the process outlined in the [OpenJDK Vulnerability Policy](https://openjdk.org/groups/vulnerability/report) to disclose vulnerabilities in the OpenJDK codebase.
```

Comment:
s/OpenJDK/JDK/ in the title and immediately preceding “codebase”, please. “OpenJDK” is the name of a community; “JDK” is the name of a body of code.

Having done that, you could simplify “in the JDK codebase” simply to “in the JDK”.

Reply:
Updated PTAL